src/test/scala/io/druid/indexer/spark/TestSparkDruidIndexer.scala

```diff
@@ -114,13 +113,14 @@ class TestSparkDruidIndexer extends FlatSpec with Matchers {
           val qindex = new QueryableIndexIndexableAdapter(index)
           qindex.getDimensionNames.asScala.toSet should
             equal(
-              dataSchema.getParser.getParseSpec.getDimensionsSpec.getDimensions.asScala.toSet --
+              dataSchema.getParser.getParseSpec.getDimensionsSpec.getDimensionNames.asScala.toSet --
                 dataSchema.getParser.getParseSpec.getDimensionsSpec.getDimensionExclusions.asScala.toSet
             )
           for (dimension <- qindex.getDimensionNames.iterator().asScala) {
-            val dimVal = qindex.getDimValueLookup(dimension).asScala
+            val dimVal = qindex.getDimValueLookup(dimension).asScala.map(Strings.nullToEmpty)
             dimVal should not be 'Empty
             for (dv <- dimVal) {
+              dv should not be null
```

Comment:
change in null behavior. Prior behavior had no null value in the iterable over strings, but current version has a null value in the dimension value iterable. I thiiiink it is expected